src/server/page.tsx

```diff
@@ -121,14 +125,16 @@ export const getPage: Koa.Middleware = async (ctx) => {
     <StaticRouter location={ctx.request.originalUrl} context={routerContext}>
       <HelmetProvider context={helmetContext}>
         <ApolloProvider client={apolloClient}>
-          <MobileContext.Provider
-            value={isMobile({
-              ua: ctx.req.headers['user-agent'],
-              tablet: true,
-            })}
-          >
-            <App session={session} dontPanicSession={dontPanicSession} />
-          </MobileContext.Provider>
+          <LegacyApolloProvider client={apolloClient}>
```

Comment:
👵

Reply:
Some problem with using `react-apollo` and the `@apollo/` packages at the same time 🙄 https://github.com/apollographql/react-apollo/issues/3381#issuecomment-550330189